string speech_syn_goal
---
bool speech_syn_result
---
bool speech_syn_feedback

